apps/metrics/templates/metrics/klass_metrics.html

```diff
@@ -236,12 +236,11 @@ <h5>Team Submission Times</h5>
         function dropdown_setup() {
             CHAGRADE.api.get_klass({{ klass.pk }})
                 .done(klass => {
```

Comment:
Refactored into a single call to map to create array.

Reply:
👍